src/map/skill.c

```diff
@@ -18984,7 +18984,7 @@ void skill_init_unit_layout (void)
 
 void skill_init_unit_layout_unknown(int skill_idx)
 {
-	ShowError("unknown unit layout at skill %d\n", skill_idx);
+	ShowError("unknown unit layout at skill %d\n", skill->dbs->db[skill_idx].nameid);
```

Comment:
How about adding an assertion (to ensure that skill_idx is between 0 and MAX_SKILL_DB) before this call? (just in case the function is called in third party code, passing an invalid index)